exports.auth = async (res, req, next) => {
  try {
  } catch (err) {
    console.log(err);
    res.status(500).send("Server Error");
  }
};
